src/ui/base-page.js

```diff
@@ -164,6 +171,84 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.
         }
     });
 
+    // mix-in grade to include login/logout controls on the page
+    fluid.defaults("sjrk.storyTelling.base.page.withAuthorControls", {
```

Comment:
This component is more than holding author account information. It's this plus the logout functionality. The current component structure is a bit messy in the way that:
```
sjrk.storyTelling.base.page.withAuthorControls = auth info + logout functionality
sjrk.storyTelling.base.page.login = sjrk.storyTelling.base.page.withAuthorControls + login functionality
```

The login component seems serving the entire account management with both login and logout. What I'm thinking is:
1. Rename the use of `login` for both the component name and the file name `base-page-login.js` to `accountManager`;
2. Move this component into the `accountManager` file;
3. Either create a single `accountManager` component containing login, logout, author account management functionalities, or split 3 functionalities into 3 separate components then assemble.

Reply:
I agree that the current scheme is a bit clunky and not entirely accurate, and I was having trouble coming up with an appropriate name for this mix-in grade that didn't seem too much like a boolean variable type name like `hasAccountControls` or `isLoginEnabled`. The `ui` called `authorControls` contains the user greeting, the logout button and the link to the Login and Sign Up pages, hence why this mix-in is a page "with author controls". Those elements should be included on every page for a site that has accounts enabled.

If we implement this proposed setup as I understand it, then we would need to include the code for the login process on any page that we want to have the authorControls included on, which would increase the payload.

The structure I chose was primarily because the login elements only need to be present on the actual login page (under the current design, at least), since it is its own discrete HTML page. I feel that it might be misleading for the page that an author would go to for logging in to (and its underlying `ui` component) be called `accountManager`, as there isn't otherwise any account management happening.

Reply:
Please add this reason that results in the current component design as a code comment. Thanks.